modules/reindex/src/internalClusterTest/java/org/elasticsearch/migration/AbstractFeatureMigrationIntegTest.java

```diff
@@ -66,6 +68,9 @@ public abstract class AbstractFeatureMigrationIntegTest extends ESIntegTestCase
     static final int INDEX_DOC_COUNT = 100; // arbitrarily chosen
     static final int INTERNAL_MANAGED_FLAG_VALUE = 1;
     public static final Version NEEDS_UPGRADE_VERSION = TransportGetFeatureUpgradeStatusAction.NO_UPGRADE_REQUIRED_VERSION.previousMajor();
```

Comment:
Do we still need this Version field, or are we able to move all `NEEDS_UPGRADE_VERSION` usages to `NEEDS_UPGRADE_INDEX_VERSION` and we can remove it?

Reply:
The release `Version` is independent of the `IndexVersion`, so we need both here

Reply:
Good to know, thanks